R/REST_helpers.R

```diff
@@ -92,6 +92,14 @@ sbtools_PUT <- function(url, body, ..., session) {
 #' @keywords internal
 sbtools_DELETE <- function(url, ..., session) {
 	check_session(session)
+	
+	uid <- tryCatch(user_id(session = session), 
```

Comment:
The first time you hist this, I am now caching the user_id in the pkg.env so it's free for every subsequent delete after the first one.